examples/blog/src/pages/posts/introducing-astro.md

```diff
@@ -62,9 +60,7 @@ This new approach to web architecture is called [islands architecture](https://j
 
 ## Embracing the Pit of Success
 
-<BlockQuote author="Jeff Atwood" source="Falling Into The Pit of Success" sourceHref="https://blog.codinghorror.com/falling-into-the-pit-of-success/">
-  A well-designed system makes it easy to do the right things and annoying (but not impossible) to do the wrong things
-</BlockQuote>
+> A well-designed system makes it easy to do the right things and annoying (but not impossible) to do the wrong things<div class="source"><p>– Jeff Atwood</p>[Falling Into The Pit of Success](https://blog.codinghorror.com/falling-into-the-pit-of-success/)</div>
```

Comment:
wait I'm confused, `BlockQuote` and `Note` are meant to look different, how are you able to achieve that if both are using the same `>`?

Reply:
I didn’t do that as I thought keeping it all markdown except for the blockquote source was better. If we want to keep the different styling, we could wrap it with a `<div class="note">` and add styles for that class.

Reply:
Okay, I think this makes sense. No need to make the example more complex than it needs to be.